distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -15,19 +15,19 @@ The Content Directory Monitor is installed by default with a standard installati
 
 === Configuring the Content Directory Monitor
 
+.Content Directory Monitor Permissions
+[IMPORTANT]
+====
+When configuring a Content Directory Monitor, make sure to set permissions on the new directory to allow ${branding} to access it. Setting permissions should be done *before* configuring a CDM. See the CDM section in ${ddf.home}/security/configurations.policy for in-depth instructions on configuring permissions.
```

Comment:
That's definitely possible, I don't think likely though. I put them in this file because I thought it would be easier for the admin rather than having to bounce back and forth between documentation and the policy file, and then just pointed to the file from the docs. I didn't want to duplicate the instructions either, but that is an option. Or I could just put them in the docs and have only the CDM policy section in the policy file, which would definitely be safer in preserving the documentation.

Reply:
I would rather preserve the documentation and have the policy file point to the docs.